pkg/client/client.go

```diff
@@ -43,6 +44,9 @@ func isDebugMode() error {
 	if debugEnv == "on" {
 		return nil
 	}
+	if viper.GetBool("Debug") {
```

Comment:
Why are we using this flag?

Reply:
if we use it in moccli, then it should remain there. We cannot extend its use into sdk